linera-execution/src/evm/revm.rs

```diff
@@ -324,12 +324,15 @@ enum PrecompileTag {
     Service(ServicePrecompileTag),
 }
 
-fn get_precompile_output(output: Vec<u8>) -> Result<Option<InterpreterResult>, String> {
+fn get_precompile_output(
+    output: Vec<u8>,
+    gas_limit: u64,
+) -> Result<Option<InterpreterResult>, String> {
     // The gas usage is set to zero since the proper accounting is done
     // by the called application
     let output = Bytes::copy_from_slice(&output);
     let result = InstructionResult::default();
-    let gas = Gas::new(0);
+    let gas = Gas::new(gas_limit);
```

Comment:
Comment above is outdated - it says gas is set to 0. Update it - but not remove - so that it matches the current behavior.

Reply:
Thank you. Corrected.